ethereum/alchemy_paymasterSupportedEntryPoints.yaml

```diff
@@ -0,0 +1,57 @@
+openapi: 3.1.0
+info:
+  title: alchemy_paymasterSupportedEntryPoints - Ethereum
```

Comment:
talked in slack but let's move this to it's own folder

Reply:
Done ✅